Source/WebCore/css/CSSCounterStyle.cpp

```diff
@@ -41,9 +41,14 @@ namespace WebCore {
 String CSSCounterStyle::counterForSystemCyclic(int value) const
 {
     auto amountOfSymbols = symbols().size();
-    int symbolIndex = (value - 1) % amountOfSymbols;
-    if (symbolIndex < 0)
-        symbolIndex += amountOfSymbols;
+    if (amountOfSymbols < 1) {
+        ASSERT_NOT_REACHED();
+        return { };
+    }
+
+    unsigned symbolIndex { 0 };
+    symbolIndex = static_cast<unsigned>(value > 0 ? value : value + amountOfSymbols);
```

Comment:
```suggestion
    unsigned symbolIndex = static_cast<unsigned>(value > 0 ? value : value + amountOfSymbols);
```

A comment explaining this is to avoid substracting 1 from the minimum integer would be nice.